docs/open-telemetry.asciidoc

```diff
@@ -410,3 +410,9 @@ Here is an example of AWS Lambda Node.js function managed with Terraform and the
 ===== OpenTelemetry logs
 
 * OpenTelemetry logs are supported with **beta support** from 8.0 https://github.com/elastic/apm-server/pull/6768[#6768].
+
+[float]
+[[open-telemetry-otlp-limitations]]
+===== OpenTelemetry Line Protocol (OTLP)
+
+* The [OTLP/HTTP protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlphttp) is not yet supported by Elastic. The [OTLP/gRPC protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlpgrpc) MUST be used. If architecture contraints require to use HTTP rather than gRPC on some segments of the network, please consider deploying an OpenTelemetry Collector to convert OTLP/HTTP to OTLP/gRPC to send those signals to Elastic.
```

Comment:
> The [OTLP/gRPC protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlpgrpc) MUST be used.

I understand that you want to stress that this is the only option, but the *MUST* reads a bit harsh to me. How about something like 
_Use the [OTLP/gRPC protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlpgrpc)_.

---

typo: architecture contraints/architecture constraints

Reply:
Thanks @simitt , please see the updated phrasing, I adopted all your suggestions.